pywr-core/src/models/simple.rs

```diff
@@ -88,6 +88,32 @@ impl From<ModelRunError> for PyErr {
     }
 }
 
+/// The results of a model run.
+///
+/// Only recorders which produced a result will be present.
+#[cfg_attr(feature = "pyo3", pyclass)]
+#[derive(Clone)]
+pub struct ModelResult {
+    network_result: NetworkResult,
+}
+
+impl ModelResult {
+    /// Get a reference to the results map.
+    pub fn network_result(&self) -> &NetworkResult {
+        &self.network_result
+    }
+}
+
+#[cfg(feature = "pyo3")]
+#[pymethods]
+impl ModelResult {
```

Comment:
a `__repr__` method, showing some basic information on scenarios/speed/etc..., would be useful, especially for notebooks